packages/social-previews/src/tumblr-preview/default-link-preview.tsx

```diff
@@ -0,0 +1,52 @@
+import { __ } from '@wordpress/i18n';
+import { baseDomain } from '../helpers';
+import { tumblrTitle, tumblrDescription } from './helpers';
+import TumblrPostActions from './post/actions';
+import TumblrPostHeader from './post/header';
+import type { TumblrPreviewProps } from './types';
+
+const TumblrDefaultLinkPreview: React.FC< TumblrPreviewProps > = ( {
+	title,
+	description,
+	image,
+	user,
+	url,
+} ) => {
+	const avatarUrl = user?.avatarUrl;
+
+	return (
+		<div className="tumblr-preview__post">
+			{ avatarUrl && <img className="tumblr-preview__avatar" src={ avatarUrl } alt="" /> }
+			<div className="tumblr-preview__card">
+				<TumblrPostHeader user={ user } />
+				<div className="tumblr-preview__window">
+					{ image && (
+						<div className="tumblr-preview__window-top">
+							<div className="tumblr-preview__overlay">
```

Comment:
I think this might be a miscommunication from the designs. The link preview would look like this if the OG image is set to one created with Social Image Generator. It shouldn't need to be implemented here, and we can pull in the post's OG image.

Reply:
This is a generic npm package and does not know where the image comes from. It's the responsibility of the consumer component to supply the image.

Reply:
Exactly, so we shouldn't be overlaying title text on this image. If text needs to be on the image, then that should be handled by whatever is using the this component.

Reply:
How should we approach it then?

Reply:
I just tested the linkedin preview and it looks like that without SIG.